src/plugins/jsx/index.js

```diff
@@ -345,6 +345,15 @@ pp.jsxParseElementAt = function(startPos, startLoc) {
           break;
 
         case tt.braceL:
+          if (this.lookahead().type === tt.ellipsis) {
+            let node = this.startNode();
+            this.next();
+            this.next();
+            node.expression = this.parseExpression();
+            this.expect(tt.braceR);
+            children.push(this.finishNode(node, "JSXSpreadChild"));
```

Comment:
`JSXSpreadChild` would need to be added to [babel-types](https://github.com/babel/babel/blob/master/packages/babel-types/src/definitions/jsx.js) as well and also to [babel-generator](https://github.com/babel/babel/blob/master/packages/babel-generator/src/generators/jsx.js)